auto_submit/lib/service/github_service.dart

```diff
@@ -29,6 +29,36 @@ class GithubService {
     return await github.repositories.getCommit(slug, sha);
   }
 
+  Future<List<PullRequestFile>> getPullRequestFiles(RepositorySlug slug, PullRequest pullRequest) async {
+    int? pullRequestId = pullRequest.number;
+    List<PullRequestFile> listPullRequestFiles = [];
```

Comment:
Use `final` keyword for variables that are not expected to change in runtime.

Reply:
Done.